code/modules/martial_arts/combos/judo/goldenblast.dm

```diff
@@ -6,6 +6,8 @@
 	combo_text_override = "Help, Disarm, Help, Grab, Disarm, Disarm, Grab, Help, Disarm, Disarm, Grab, Help"
 
 /datum/martial_combo/judo/goldenblast/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(IS_HORIZONTAL(user)) // Not performing Golden Blast being prone
+		return MARTIAL_COMBO_FAIL
```

Comment:
This is not part of #26053 and should not be included in a fix for it.

Reply:
> tweak: Armbar and Wheel Throw / Ground Pin can now be used on the ground.

This is what #25659 says. In my PR i only give an ability to finally use the armbar and a wheel throw being prone. But if we want to use a golden blast as well, okay, I'll remove that part of a code.

Reply:
This PR is labelled as a fix for #26053. Golden Blast is not mentioned, and changes to it shouldn't be included in a pure fix PR. Doing so would turn it into an actual balance PR (besides, golden blast is long and complex enough that it doesn't really need a nerf, successfully pulling it off in actual combat is a very difficult proposition compared to doing it "properly").

Outside of that nit, the code is functional.